projects/driver_display/inc/driver_display_brightness_config.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+// Configuration module for the brightness module and photo sensor calibration module
+// Defines screen constants and update periods to control frequency of brightness changes
+
+#include "driver_display_brightness.h"
+
+// Frequency for the screens (All the same frequency)
+#define DRIVER_DISPLAY_CONFIG_SCREEN_FREQ_HZ 30000
```

Comment:
If your PWM module is currently operating at 1Mhz, at 30kHz you have a pulse width of 33us. This means that you can't actually get 0 to 100% duty cycle.